### PR TITLE
Feature/LEWW-70 | updating landing (es/en), how-to/organization, other sections + IaC Lib renamed

### PR DESCRIPTION
**Pull request by exequielrafaela**
### Commits on Jun 18, 2021
- @exequielrafaela - LEWW-70 | updating /user-guide/organization sections - 611e48e
- @exequielrafaela - LEWW-70 | Adding new landing and sections figures - 65c98f2
- @exequielrafaela - LEWW-70 | updating name from "DevOps Automation Code Library" to "Infrastructure as Code (IaC) library" - b704f0e
- @exequielrafaela - LEWW-70 | several eng/spa landing updates. - 72aa728
- @exequielrafaela - LEWW-70 |new how-it-works sub-sections + varios sections updated - a455244

## Why?
- https://github.com/binbashar/le-ref-architecture-doc/issues/66
- https://github.com/binbashar/le-tf-infra-aws/issues/47
- BIZDEV-43 | [commercial] presentation improvements

### CODE REVIEW

File: docs/how-it-works/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Projects’s AWS infrastructure.
 
 Reference AWS Cloud Solutions architecture designed under optimal configs for the most
 popular modern web and mobile applications needs based on AWS [“Well Architected Framework”](../work-with-us/support.md).
-With it's complete [**Leverage DevOps Automation Code Library**](./code-library/code-library.md) to rapidly implement it
+With it's complete [**Infrastructure as Code (IaC) Library**](./code-library/code-library.md) to rapidly implement it
```
Comment (diego-ojeda-binbash) [inline docs/how-it-works/index.md:32]: I think it should say "With its complete" or maybe even "With a complete" (although honestly I think that sounds a lot like the way we speak in Argentina, can't remember expressions like that in English).

File: docs/user-guide/base-workflow/repo-le-dev-makefiles.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 In order to get the full automated potential of the
-[Binbash Leverage DevOps Automation Code Library](https://leverage.binbash.com.ar/how-it-works/code-library/code-library/) 
+[Binbash Leverage Infrastructure as Code (IaC) Library](https://leverage.binbash.com.ar/how-it-works/code-library/code-library/) 
 you should initialize all the necessary helper **Makefiles**.
```
Comment (diego-ojeda-binbash) [inline docs/user-guide/base-workflow/repo-le-dev-makefiles.md:7]: Should we mark this as deprecated?

File: docs/user-guide/organization/organization-init.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    :books: [**documentation:** organization](../../how-it-works/organization/organization.md)
+    
+    :books: [**documentation:** organization accounts](../../how-it-works/organization/accounts.md)
+
+## User guide
+
+### Pre-requisites
+
+You'll need an email to [create and register your AWS Organization Root Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+For this purpose we recommend to avoid using an individual nominated email user.
```
Comment (diego-ojeda-binbash) [inline docs/user-guide/organization/organization-init.md:13]: The word "nominated" sounds wrong. How about "We recommend that you do not use a personal email account for this."?

**Review by diego-ojeda-binbash — APPROVED** [review]

File: docs/how-it-works/index.md (hunk 1 above)
Comment (exequielrafaela) [reply docs/how-it-works/index.md:32]: @diego-ojeda-binbash OJ may be we could use "With its full". What do you think? Do you have any other alternatives or rephrasing that could fit here?

File: docs/user-guide/base-workflow/repo-le-dev-makefiles.md (hunk 2 above)
Comment (exequielrafaela) [reply docs/user-guide/base-workflow/repo-le-dev-makefiles.md:7]: @diego-ojeda-binbash OJ yes, we should definitely deprecate this. I'm seriously considering  the option to remove the `Makefiles` from the https://github.com/binbashar/le-tf-infra-aws repo.

File: docs/user-guide/organization/organization-init.md (hunk 3 above)
Comment (exequielrafaela) [reply docs/user-guide/organization/organization-init.md:13]: @diego-ojeda-binbash OJ thanks :) I'll update this 🤝 ✅

File: docs/user-guide/base-workflow/repo-le-dev-makefiles.md (hunk 2 above)
Comment (diego-ojeda-binbash) [reply docs/user-guide/base-workflow/repo-le-dev-makefiles.md:7]: @exequielrafaela My comment is just about marking this doc as deprecated with a note that explains why. The actual Makefiles can be addressed separate from this maybe?

File: docs/how-it-works/index.md (hunk 1 above)
Comment (diego-ojeda-binbash) [reply docs/how-it-works/index.md:32]: @exequielrafaela Can't think of anything 😅 . For now how about fixing the "it's" and later on we can think of alternatives? 😄

File: docs/user-guide/base-workflow/repo-le-dev-makefiles.md (hunk 2 above)
Comment (exequielrafaela) [reply docs/user-guide/base-workflow/repo-le-dev-makefiles.md:7]: @diego-ojeda-binbash OJ Absolutely, by considering the removal of the Makefiles, I meant that I got your point here 💪🏼

Comment (exequielrafaela) [reply docs/user-guide/base-workflow/repo-le-dev-makefiles.md:7]: @diego-ojeda-binbash We still don't have the leverage cli ansible module migrated though. So I'll add a comment in the doc till we completely replace the entry.

**Comment by exequielrafaela** [comment]
@diego-ojeda-binbash I've addressed the improvements suggested in your comments here:

- @exequielrafaela - LEWW-70 | improving eks convention support info - 720dfdf
- @exequielrafaela - LEWW-70 | introducing leverage cli + sintaxt improvements - 76ef167

**Pull request merged**